crates/bevy_window/src/window.rs

```diff
@@ -1213,8 +1226,19 @@ pub struct ClosingWindow;
 
 #[cfg(test)]
 mod tests {
+    use bevy_ecs::entity::MapEntities;
+
     use super::*;
 
+    // Compile-time assertion that a type implements MapEntitiesMut
```

Comment:
Do we need this test?

Reply:
Probably not - the ReflectMapEntities derive actually covers it pretty well.